### ISSUE TITLE
Find a way to reference current season

### ISSUE THREAD

**Issue by cmw0513** [cmw0513/scorecard#59]
should be either season in progress, or recently finished season before a new season begins

**Comment by lienvdsteen** [comment]
Do we always need a current season? Why?

**Comment by cmw0513** [comment]
I don't think we need to show the stats for all the seasons on the player profile for example, or the games schedule from past seasons. Often we just want to show the current season, which may not be the last one created (if they're getting ready for the next season)

**Comment by lienvdsteen** [comment]
Yeah you're right! So either the current season or the last season they just finished

**Comment by cmw0513** [comment]
Right now we have a current_season method for team that returns the league's last season
